pkgs/build-support/cc-wrapper/default.nix

```diff
@@ -295,6 +295,10 @@ stdenv.mkDerivation {
       hardening_unsupported_flags+=" stackprotector fortify"
     ''
 
+    + optionalString targetPlatform.isx86_32 ''
+      hardening_unsupported_flags+=" stackprotector"
```

Comment:
This is temporary I assume?

Reply:
Maybe... binutils and musl seemed to get stuck. I don't think anyone supports i686 very well any more.

Reply:
that combination or in isolation? `with targetPlatform.isx86_32 && targetPlatform.isMusl` I'd be more comfortable with that.